emails/models.py

```diff
@@ -708,6 +713,15 @@ def profile(self):
     def owner_has_premium(self):
         return self.profile.has_premium
 
+    def increment_num_replied(self):
+        address = self.relay_address or self.domain_address
+        address.num_replied += 1
+        address.last_used_at = datetime.now(timezone.utc)
```

Comment:
praise: I'm going to use this instead of `datetime.now(tz=timezone.utc)`, saves 3 characters!